.travis.yml

```diff
@@ -2,6 +2,8 @@ language: node_js
 node_js:
   - "lts/*"
   - "node"
+after_script:
+- "npm install coveralls && npm run deploy-coverage"
```

Comment:
Since this dependency is only for Travis to publish the report to Coveralls, it's not needed to add it in devDependencies. And I've remove the line from package.json.